core/templates/dev/head/pages/exploration_player/AnswerFeedbackPairDirective.js

```diff
@@ -66,6 +66,10 @@ oppia.directive('answerFeedbackPair', [
             }
             return shortAnswerHtml;
           };
+
+          $scope.getRequestForHint = function() {
+            return $scope.data.learnerAnswer;
```

Comment:
I have an idea -- can we have a single list containing answer/feedback pairs as well as request/hint pairs? request/hint pairs will have an additional isHintRequest boolean value that can be used to identify. This is to maintain the right sequence. To add a new request/hint we could have different methods (like addNewHintRequest/addNewHint).